lms/routes.py

```diff
@@ -1,4 +1,4 @@
-def includeme(config):
+def includeme(config):  # pylint:disable=too-many-statements
```

Comment:
We could make `lms/routes/admin.py`. 

I rather split it in a follow up PR.

Reply:
:shrug: This has the advantage of being a single place to look?